### PR TITLE
Update migration docs with Console feature: Import end devices

### PR DESCRIPTION
**Pull request by laurensslats**
#### Summary
Adding the Console feature for importing end devices

#### Changes
- Added documentation on importing end devices in both **Migrating from Third Party Networks** and **Migrating from V2**
- Added a `Console` / `CLI` tab for documentation 

#### Testing
Please shed your light on the documentation.

##### Regressions
n/a 

#### Notes for Reviewers
No major changes, just quickly look at both documentation pages.

#### Checklist
- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Compatibility: The changes are backwards compatible with existing API, storage, configuration and CLI, according to the compatibility commitments in `README.md` for the chosen target branch.
- [x] Documentation: Relevant documentation is added or updated.
- [ ] Changelog: Significant features, behavior changes, deprecations and fixes are added to `CHANGELOG.md`.
- [ ] Commits: Commit messages follow guidelines in `CONTRIBUTING.md`, there are no fixup commits left.

### CODE REVIEW

**Comment by laurensslats** [comment]
@neoaggelos would love to add this to the next release. A prompt review would be appreciated.

**Review by neoaggelos — COMMENTED** [review]
Content seems OK. Two questions:

- Why is gateway migration included in this PR?
- "Importing End Devices" is not strictly related to the migrating guides. It is also not a good idea to have duplicate docs (in migrating-from-v2 and migrating-from-networks).

I would suggest that `Importing End Devices` is extracted into a separate guide, and the migration guides point to that guide instead.

**Comment by laurensslats** [comment]
> Why is gateway migration included in this PR?

It shouldn't as it's already part of a different PR, removed it.

> "Importing End Devices" is not strictly related to the migrating guides. It is also not a good idea to have duplicate docs (in migrating-from-v2 and migrating-from-networks).

Removed it from this documentation and referenced the doc on `/getting-started/migrating-from-networks/device-json/`

File: doc/content/getting-started/migrating-from-v2/mqtt-api-comparison.md [hunk 1]
```diff
@@ -77,4 +77,4 @@ weight: 70
 
 [TTI V2 MQTT API Documentation](https://www.thethingsnetwork.org/docs/applications/mqtt/api.html)
 
-[TTES V3 MQTT API Documentation](https://enterprise.thethingsstack.io/v3.8.1/integrations/mqtt/)
+[TTS V3 MQTT API Documentation](https://enterprise.thethingsstack.io/v3.8.1/integrations/mqtt/)
```
Comment (neoaggelos) [inline doc/content/getting-started/migrating-from-v2/mqtt-api-comparison.md]: We have dropped version number from documentation links right? Also, should this not be a ref instead?

File: doc/content/getting-started/migrating-from-networks/device-json.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-title: "Create Device.json"
+title: "Create devices.json"
```
Comment (neoaggelos) [inline doc/content/getting-started/migrating-from-networks/device-json.md]: Probably change to a title that describes the process instead, like "Export End Devices"

File: doc/content/getting-started/migrating-from-networks/import-devices.md [hunk 3]
```diff
@@ -3,20 +3,52 @@ title: Import End Devices in The Things Stack
...
+
+Open the application you created and click the button **Import end devices**
+
+{{< figure src="../import-end-devices.png" alt="import devices" >}}
+
+Select `The Things Stack JSON` as the **Format** and upload the `devices.json` file.
+
+{{< figure src="../upload-file.png" alt="upload devices.json file" >}}
+
+Wait for the devices to be successfully imported. In case any device fails, you see a relevant error message in the console.
```
Comment (neoaggelos) [inline doc/content/getting-started/migrating-from-networks/import-devices.md]: Nit; we refer to them as "end devices" in the rest of the documentation, so better do this here, and in other places also.

**Review by neoaggelos — COMMENTED** [review]
Getting there! Left some small comments

File: doc/content/getting-started/migrating-from-networks/import-devices.md [hunk 4]
```diff
@@ -3,20 +3,52 @@ title: Import End Devices in The Things Stack
...
+
+{{< figure src="../operation-finished.png" alt="import finished" >}}
 
-To import your devices, you need an Application in {{% tts %}}. This can be done by following instructions for [Creating an Application]({{< ref "integrations/adding-applications" >}}).
+If the import was successful, your devices is added to the list of end-devices in your application.
```
Comment (bafonins) [inline doc/content/getting-started/migrating-from-networks/import-devices.md]:
```suggestion
If the import was successful, your devices are added to the list of end-devices in your application.
```

**Comment by laurensslats** [comment]
Processed the latest feedback, ready to merge. 
@benolayinka can you move this to the [LoRaWAN Stack Docs](https://github.com/TheThingsIndustries/lorawan-stack-docs) repo?

**Comment by benolayinka** [comment]
moved to new repo https://github.com/TheThingsIndustries/lorawan-stack-docs/pull/20

**Pull request closed without merging**